apps/pdc-frontend/src/components/OpenFormsEmbed/OpenFormsEmbed.tsx

```diff
@@ -17,12 +19,12 @@ export const OpenFormsEmbed = ({ nonce, basePath, slug, apiUrl, sdkUrl, cssUrl,
   const id = useId();
 
   return (
-    <>
+    <div className="utrecht-html">
```

Comment:
Dit lijkt me wel een beetje overkill, omdat de NL Design System CSS al wordt meegeleverd. Ik maak me zorgen dat het tot interference en moeilijk te vinden bugs leidt.

Kunnen we niet opzoeken welke class name de wrapper is van de rich text blokken in Open Formulieren en dan SCSS te gebruiken?

```scss
.of-some-wrapper {
  @include utrecht-html;
}
```